signer/local_cosigner.go

```diff
@@ -79,8 +76,7 @@ func (cosigner *LocalCosigner) SaveLastSignedState(signState SignStateConsensus)
 // GetID returns the id of the cosigner, via the localsigner getter
 // Implements the Cosigner interface from Cosigner.go
 func (cosigner *LocalCosigner) GetID() int {
-	id, _ := cosigner.localsigner.GetID()
-	return id
+	return cosigner.id
```

Comment:
Hmmm... So the id of the cosigner or signer here?
Should it always be the same?

Reply:
Yes they should always be the same.